transport_plugins/bled112/iotile_transport_bled112/bled112_cmd_co.py

```diff
@@ -0,0 +1,843 @@
+from collections import namedtuple
+import time
+import struct
+import threading
+import logging
+import functools
+from queue import Empty
+from iotile.core.utilities.packed import unpack
+from iotile.core.utilities.async_tools import OperationManager, SharedLoop
+from iotile.core.exceptions import HardwareError
+from .tilebus import *
+from .bgapi_structures import process_gatt_service, process_attribute, process_read_handle, process_notification
+from .bgapi_structures import parse_characteristic_declaration
+from .async_packet import InternalTimeoutError, DeviceNotConfiguredError
+
+BGAPIPacket = namedtuple("BGAPIPacket", ["is_event", "command_class", "command", "payload"])
+
```

Comment:
Do we also need a license in this file? I see them some places and not others

Reply:
Negative.  A license header was only added for code that existed before Arch and is maintained as that file is edited to track any potentially derivative code.  `bled112_cmd.py` was post arch and `bled112_cmd_co.py` is the start of its replacement so it doesn't need one because it's based on `bled112_cmd` that didn't need one.